visidata/loaders/fixed_width.py

```diff
@@ -38,7 +38,7 @@ def columnize(rows):
     # collapse fields
     for i in allNonspaces:
         if i > prev+1:
-            yield colstart, i
+            yield colstart, prev+1 #2255
```

Comment:
This could also be:

``` python
            yield colstart, i-1 #2255
```

`prev+1` collapses multiple spaces between columns.
`i-1` assumes only a single space is the column separator, and keeps the remaining spaces as data.

For example, given the following fixed width data with three spaces between the columns:

```
column_1    column_2
one         two
```

`prev+1` would be saved as the following CSV. (Separating spaces are removed, but padding spaces are retained):

```
column_1,column_2
one     ,two
```

`i-1` would be saved as the following CSV. (One separating space is removed. All others are retained.):

```
column_1  ,column_2
one       ,two
```

The current version of the code with `prev+1` is my personal preference, but open to guidance if the `i-1` version better matches the intent of 'fixed width'.

Reply:
Note, it must be `i-1` rather than `i` to avoid adding an extra space between the columns each round trip.

Reply:
Okay, I prefer `prev+1` too.  Thanks for the explanation.